### ISSUE TITLE
Build fails with latest jdt

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-xtend#587]
http://services.typefox.io/open-source/jenkins/job/xtext-xtend/job/master/708/console

```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:1.2.0:compile (default-compile) on project org.eclipse.xtend.ide: Compilation failure: Compilation failure:
[ERROR] /var/jenkins_home/workspace/xtext-xtend_master-CZ6MQ7K3F54I6ECVAJOY5S5BWV6SPAS3CFPK73ZFQFHMBDDXMCYA/org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/wizards/AbstractNewXtendElementWizardPage.java:[29]
[ERROR] import org.eclipse.jdt.internal.corext.codemanipulation.StubUtility;
[ERROR] ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[ERROR] The import org.eclipse.jdt.internal.corext.codemanipulation.StubUtility cannot be resolved
[ERROR] /var/jenkins_home/workspace/xtext-xtend_master-CZ6MQ7K3F54I6ECVAJOY5S5BWV6SPAS3CFPK73ZFQFHMBDDXMCYA/org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/wizards/AbstractNewXtendElementWizardPage.java:[93]
[ERROR] return StubUtility.getCompilationUnitContent(compilationUnit, packageDeclaration, fileComment, typeComment, typeContent, lineSeparator);
```

**Comment by cdietrich** [comment]
https://github.com/eclipse/eclipse.jdt.ui/commit/bc3bc858da1855f31cee9566912859faac05819e

**Comment by cdietrich** [comment]
again we are hit by using internal api

**Comment by cdietrich** [comment]
Fixed in 2.16
